identity-service/src/relay/txRelay.js

```diff
@@ -106,6 +111,25 @@ const sendTransactionInternal = async (req, web3, txProps, reqBodySHA) => {
     contractRegistryKey.charAt(0).toUpperCase() + contractRegistryKey.slice(1) // uppercase the first letter
   const decodedABI = AudiusABIDecoder.decodeMethod(contractName, encodedABI)
 
+  // Rate limit replica set reconfiguration transactions
+  // A reconfiguration (as opposed to a first time selection) will have an
+  // _oldPrimaryId value of "0"
+  const isReplicaSetTransaction = decodedABI.name === 'updateReplicaSet'
+  if (isReplicaSetTransaction) {
+    const isFirstReplicaSetConfig = decodedABI.params.find(
+      (param) => param.name === '_oldPrimaryId' && param.value === '0'
+    )
+    if (!isFirstReplicaSetConfig) {
+      transactionRateLimiter.updateReplicaSetReconfiguration += 1
+      if (
+        transactionRateLimiter.updateReplicaSetReconfiguration >
+        UPDATE_REPLICA_SET_RECONFIGURATION_LIMIT
+      ) {
+        throw new Error('Is not first replica set config')
```

Comment:
prob should say `updateReplicaSet rate limit reached`

Reply:
updated